avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,25 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, options=None, sudo=False):
+        """Check packet loss that occurs during ping.
+
+        Function returns True for 0% packet loss and False
+        if packet loss occurs.
+
+        :param peer_ip: Peer IP address (IPv4 or IPv6)
+        :param options: ping command options such as -c, -f. Default is None
```

Comment:
What is the type of `options`? Python developers are not used to have a list of "things" inside a string. Strings, inside python, should not be used to pass list of "things", differently from bash scripts. We prefer using a "list" for lists.

My suggestions here:
  1. Convert options to accept a list, the same way we do with "args" when running a subcommand.
  2. Document here with `:type options: list`.
  3. Do the same with the other arguments.